src/components/navigation-header/components/home-page-content/index.tsx

```diff
@@ -36,6 +36,15 @@ const HomePageHeaderContent = () => {
 		}
 	})
 
+	// Construct item groups for the dropdown, avoid adding empty groups
+	const itemGroups = []
+	if (betaProductItems.length) {
+		itemGroups.push({ items: betaProductItems })
+	}
+	if (comingSoonProductItems.length) {
+		itemGroups.push({ label: 'Coming Soon', comingSoonProductItems })
+	}
```

Comment:
The `comingSoonProductItems` need to be sent as the `items` property here. 😊 Tested by removing one of the products from the `base.json` `beta_products_array`!

Reply:
Oops dang great call! ty 💖 updated ✅